docs/integration.md

```diff
@@ -47,19 +65,19 @@ The endpoint accepts a `POST` request with
 
 | Variable Name | Type                          | Description         |
 | ------------- | ----------------------------- | ------------------- |
-| id            | string                        | event id            |
-| type          | enum: [EventType](#eventtype) |
-| data          | Object                        | any additional data |
+| `id`          | String                        | event id            |
+| `type`        | Enum: [EventType](#eventtype) |
+| `data`        | Object                        | any additional data |
 
 #### EventType
 
-| Value                    | Description                                                                 |
-| ------------------------ | --------------------------------------------------------------------------- |
-| IncomingPaymentCompleted | Incoming payment is complete and doesn't accept any incoming funds anymore. |
-| IncomingPaymentExpired   | Incoming payment is expired and doesn't accept any incoming funds anymore.  |
-| OutgoingPaymentCreated   | Outgoing payment was created.                                               |
-| OutgoingPaymentCompleted | Outgoing payment is complete.                                               |
-| OutgoingPaymentFailed    | Outgoing payment failed.                                                    |
+| Value                      | Description                                                                 |
+| -------------------------- | --------------------------------------------------------------------------- |
+| `IncomingPaymentCompleted` | Incoming payment is complete and doesn't accept any incoming funds anymore. |
+| `IncomingPaymentExpired`   | Incoming payment is expired and doesn't accept any incoming funds anymore.  |
+| `OutgoingPaymentCreated`   | Outgoing payment was created.                                               |
+| `OutgoingPaymentCompleted` | Outgoing payment is complete.                                               |
+| `OutgoingPaymentFailed`    | Outgoing payment failed.                                                    |
```

Comment:
Should these actually be in a structure like: `incoming_payment.completed`? Since that is what is sent from `backend`